### PR TITLE
GEOPY-470

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-470 - Crash on geoh5 without Types**

### CODE REVIEW

File: geoh5py/io/h5_reader.py [hunk 1]
```diff
@@ -87,6 +87,9 @@ def fetch_attributes(
             for key, value in entity.attrs.items():
                 attributes["entity"][key] = value
 
+            if "Type" not in entity:
```
Comment (benk-mira) [inline geoh5py/io/h5_reader.py:90]: In the writer, you are creating a group called 'Types', but here it is just 'Type'.  Is this correct?

Comment (benk-mira) [reply geoh5py/io/h5_reader.py:90]: hmm.. Based on the tests, I'm guessing that this is correct, but I'll leave the comment in here so you can explain to me what's going on!!

Comment (domfournier) [reply geoh5py/io/h5_reader.py:90]: That's just geoh5 naming convention. "Types" is a group containing the "Type" of all objects... not my choice

Comment (benk-mira) [reply geoh5py/io/h5_reader.py:90]: gotcha

**Review by benk-mira — APPROVED** [review]
All good

**Pull request merged**